pkg/snowflake/schema.go

```diff
@@ -120,12 +120,12 @@ func (sb *SchemaBuilder) Swap(targetSchema string) string {
 
 // ChangeComment returns the SQL query that will update the comment on the schema.
 func (sb *SchemaBuilder) ChangeComment(c string) string {
-	return fmt.Sprintf(`ALTER SCHEMA %v SET COMMENT = '%v'`, sb.QualifiedName(), c)
+	return fmt.Sprintf(`ALTER SCHEMA %v SET COMMENT = '%v'`, sb.QualifiedName(), EscapeString(c))
```

Comment:
why is the other QualifiedName escaped (in RemoveComment) but not this one? I can't really tell if there is a difference or not.

Reply:
Good catch. I think qualified names should not be escaped. Or, at least, I wasn't trying to do that in this PR.